migration-guide/src/explorer-api.md

```diff
@@ -0,0 +1,2904 @@
+# explorer-api
+
+The following requests are made on the Cardano TestNet. It is also important to note that many GraphQL queries
+below are in reality much more flexible and capable than the queries given in the examples.
+
+!!! tip
+    Any GraphQL query can be run interactively in the Cardano GraphQL playground, which is accessible [here](https://cardano-graphql-testnet.daedalus-operations.com/) online.
+
+## Blocks
+
+#### [/api/blocks/pages](https://input-output-hk.github.io/cardano-rest/explorer-api/#operation/_blocksPages)
+
+Get the list of blocks, contained in pages.
+
+=== "cardano-rest"
+
+    ```console
+    $ curl "https://explorer.cardano-testnet.iohkdev.io/api/blocks/pages?page=221038"
```

Comment:
I chose to use the public testnet instance. I myself have major issues using the docker containers, so running quick queries on those is the easiest way for testing.